src/core/blockchain/celo/stats.ts

```diff
@@ -1,8 +1,229 @@
-import { GenericStats, AccountStats } from '../types/stats';
+import {
+    GenericStats,
+    IStatValueType,
+    AccountStats,
+    IValidatorCard,
+    CardActionType
+} from '../types/stats';
 import { Client } from './client';
+import { Blockchain } from '../types/blockchain';
 
 export class Stats extends GenericStats<Client> {
     public getAccountDelegateStats(): AccountStats {
```

Comment:
this needs to be async, when we will implement this we will need to do some http requests